src/components/header.tsx

```diff
@@ -1,7 +1,9 @@
 import { VNode } from "preact";
+import { getFullPath, Link } from "../router";
+// eslint-disable-next-line import/no-relative-packages
 import logoURL from "../../node_modules/identity-style-guide/dist/assets/img/login-gov-logo.svg";
```

Comment:
Out of curiosity, would it work to import it from the package name base?

```suggestion
import logoURL from "identity-style-guide/dist/assets/img/login-gov-logo.svg";
```

Reply:
vite was upset:
```
1:26:58 PM [vite] Internal server error: Missing "./dist/assets/img/login-gov-logo.svg" export in "identity-style-guide" package
  Plugin: vite:import-analysis
```